crates/assistant/src/assistant_panel.rs

```diff
@@ -2537,8 +2539,52 @@ impl ContextEditor {
                                     )
                                 }
                             };
-                            let render_trailer =
-                                |_row, _unfold, _cx: &mut WindowContext| Empty.into_any();
+                            let render_trailer = {
+                                let command = command.clone();
+                                move |row: MultiBufferRow, _unfold, cx: &mut WindowContext| {
+                                    if command.name == "rustdoc" {
```

Comment:
At some point this should probably be something exposed on the `SlashCommand` trait, but for now I just special-cased the `/rustdoc` command in here.